src/pages/projects/[index].tsx

```diff
@@ -33,7 +34,10 @@ export default function Project(): JSX.Element {
             <ProjectDetails />
             <ChatBubble />
             <Gallery />
-            <div>Previous Next Button Block</div>
+            <div className={styles.projectNav}>
+                <ProjectNav text={'Previous'} />
+                <ProjectNav text={'Next'} />
```

Comment:
I left a comment about this earlier, but how about naming to "buttonText"?

Reply:
I like it, change made.